spec/support/coronavirus_feature_steps.rb

```diff
@@ -338,7 +338,7 @@ def when_i_fill_in_the_timeline_entry_form_with_valid_data
 
   def then_i_see_a_new_timeline_entry_has_been_created
     expect(current_path).to eq("/coronavirus/landing")
-    expect(expect(page).to(have_text("Fancy title")))
```

Comment:
Oops! good spot.

Reply:
Unfortunately I think I was the perpetrator of that particular crime 🤦